tests/unit/models/transactions/test_amm_deposit.py

```diff
@@ -0,0 +1,151 @@
+from unittest import TestCase
+
+from xrpl.models.amounts import IssuedCurrencyAmount
+from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.transactions import AMMDeposit
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_AMM_HASH = "24BA86F99302CF124AB27311C831F5BFAA72C4625DDA65B7EDF346A60CC19883"
+_AMOUNT = "1000"
+_LPTOKEN_CURRENCY = "B3813FCAB4EE68B3D0D735D6849465A9113EE048"
+_LPTOKEN_ISSUER = "rH438jEAzTs5PYtV6CHZqpDpwCKQmPW9Cg"
+
+
+class TestAMMDeposit(TestCase):
+    def test_to_xrpl_lptokens(self):
+        tx = AMMDeposit(
+            account=_ACCOUNT,
+            sequence=1337,
+            amm_hash=_AMM_HASH,
+            lptokens=IssuedCurrencyAmount(
+                currency=_LPTOKEN_CURRENCY,
+                issuer=_LPTOKEN_ISSUER,
+                value=_AMOUNT,
+            ),
+        )
+        expected = {
+            "AMMHash": _AMM_HASH,
+            "Account": "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ",
+            "LPTokens": {
```

Comment:
Why is this plural if it is an object?  I realize this is a spec question.

Reply:
Waiting for Liquidity team's feedback/decision on this.

Reply:
Waiting for Liquidity team's feedback/decision on renaming it to be singular: `LPToken`.

Reply:
They agreed to rename it to `LPToken` for consistency. I just pushed this refactor.